lib/warder.rb

```diff
@@ -10,6 +10,7 @@
 require 'warder/rails_advice_runner'
 require 'warder/sandi_rules_runner'
 require 'warder/bundle_audit_runner'
+require 'warder/coffee_lint_runner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.